public/index.html

```diff
@@ -5,7 +5,8 @@
     <title>Hyper Says</title>
     <link rel="stylesheet" type="text/css" href="animate.css">
     <link rel="stylesheet" type="text/css" href="styles.css">
-    <script type="text/javascript" src="https://rawgit.com/hyperoslo/hyper-says/master/public/quotes.js"></script>
+    <script type="text/javascript" src="quotes.js"></script>
```

Comment:
Hmm, I think the reason we had this in the first place was so that we wouldn't need to redeploy every time a quote was added. At least that would make sense.